### PR TITLE
Feat/user page

### PR DESCRIPTION
**Pull request by davidalexandru1370**
1. Username is taken from route
2. The user is fetched by username and data gets displayed
3. Post button does not have functionality

### CODE REVIEW

**Comment by davidalexandru1370** [comment]
Screenshot (173)
Screenshot (174)

File: src/app/pages/profile/profile.component.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            <mat-icon
+              aria-hidden="false"
+              aria-label="Avatar"
+              role="img"
+              style="font-size: 40px; width: 45px; height: 50px"
+              >account_circle
+            </mat-icon>
+            <div style="display: flex; flex-direction: column">
+              <span style="font-weight: 800" *ngIf="user !== null">
+                {{ user!.firstName + " " + user.lastName }}
```
Comment (Laurcons) [inline src/app/pages/profile/profile.component.html]:
```suggestion
                {{ user?.firstName + " " + user.lastName }}
```
Cum e acum o sa dea crash daca e undefined... cu `?` macar doar apare string gol

**Review by Laurcons — DISMISSED** [review]
Poti baga asa cum e acum, mi se pare ok

File: src/app/pages/profile/profile.component.html (hunk 1 above)
Comment (Laurcons) [reply src/app/pages/profile/profile.component.html]: Da resolve la conversatie daca decizi sa nu implementezi asta :))

Comment (davidalexandru1370) [reply src/app/pages/profile/profile.component.html]: Am considerat a se merge doar pe un happy flow fara validari

**Review by Laurcons — APPROVED** [review]

**Pull request merged**